tracepath and ping fail to compile with musl-libc and linux-headers 4.2.0

Ran into this while cross compiling for musl. Will check to see if it works fine natively once I can.

```
make -j4 CC="x86_64-pc-linux-musl-cc" CFLAGS="-march=native -pipe -Os -g" LIBC_INCLUDE="/usr/x86_64-pc-linux-musl/include" USE_CAP=no USE_GCRYPT=no USE_IDN=no USE_NETTLE=no
fatal: Not a git repository (or any of the parent directories): .git
x86_64-pc-linux-musl-cc -march=native -pipe -Os -g -march=native -pipe -Os -g -D_GNU_SOURCE  -c ping.c    -o ping.o
x86_64-pc-linux-musl-cc -march=native -pipe -Os -g -march=native -pipe -Os -g -D_GNU_SOURCE  -c ping_common.c   -o ping_common.o
x86_64-pc-linux-musl-cc -march=native -pipe -Os -g -march=native -pipe -Os -g -D_GNU_SOURCE  -c ping6_common.c   -o ping6_common.o
x86_64-pc-linux-musl-cc -march=native -pipe -Os -g -march=native -pipe -Os -g -D_GNU_SOURCE  -c tracepath.c  -o tracepath.o
In file included from tracepath.c:17:0:
/usr/x86_64-pc-linux-musl/include/linux/errqueue.h:33:18: error: array type has incomplete element type 'struct timespec'
  struct timespec ts[3];
                  ^
Makefile:137: recipe for target 'tracepath.o' failed
make: *** [tracepath.o] Error 1
make: *** Waiting for unfinished jobs....
ping.c: In function 'main':
ping.c:289:22: error: 'SCOPE_DELIMITER' undeclared (first use in this function)
     p = strchr(addr, SCOPE_DELIMITER);
                      ^
ping.c:289:22: note: each undeclared identifier is reported only once for each function it appears in
Makefile:137: recipe for target 'ping.o' failed
make: *** [ping.o] Error 1
```

--- a/ping.h
+++ b/ping.h
@@ -57,6 +57,10 @@
 #include "in6_flowlabel.h"
 #include "SNAPSHOT.h"
 
+#ifndef SCOPE_DELIMITER
+#define SCOPE_DELIMITER '%'
+#endif
+
 #define	DEFDATALEN	(64 - 8)	/* default data length */
 
 #define	MAXWAIT		10		/* max seconds to wait for response */

--- a/tracepath.c
+++ b/tracepath.c
@@ -14,7 +14,6 @@
 #include <unistd.h>
 #include <sys/socket.h>
 #include <linux/types.h>
-#include <linux/errqueue.h>
 #include <errno.h>
 #include <string.h>
 #include <netdb.h>
@@ -22,6 +21,7 @@
 #include <netinet/in.h>
 #include <resolv.h>
 #include <sys/time.h>
+#include <linux/errqueue.h>
 #include <sys/uio.h>
 #include <arpa/inet.h>
 

